lib/mix/tasks/ecto.migrate.ex

```diff
@@ -121,6 +127,9 @@ defmodule Mix.Tasks.Ecto.Migrate do
         do: Keyword.merge(opts, log: false, log_migrations_sql: false, log_migrator_sql: false),
         else: opts
 
+    log_level = String.to_existing_atom(Keyword.get(opts, :log_level, "debug"))
+    Logger.configure(level: log_level)
```

Comment:
I wonder if it's possible for people to have had log level set beforehand, i.e. they used this task from a script or something. If that'd be possible, perhaps we should only set the level if option is given. Something like:

```suggestion
    if log_level = Keyword.get(opts, :log_level) do
      level = String.to_existing_atom(log_level)
      Logger.configure(level: log_level)
    end
```

Reply:
This is a great call yes. Updated 💯